theme/static/js/resourceLandingAjax.js

```diff
@@ -1011,33 +1109,52 @@ function initializeDatePickers(){
 }
 
 // act on spatial coverage type change
-function setFileTypeSpatialCoverageFormFields(){
-    $("#div_id_type_filetype input:radio").change(function () {
-        if ($(this).val() == "point" && $(this).attr("checked") == "checked") {
-            $("#div_id_north_filetype").show();
-            $("#div_id_east_filetype").show();
-            $("#div_id_elevation_filetype").show();
-            $("#div_id_northlimit_filetype").hide();
-            $("#div_id_eastlimit_filetype").hide();
-            $("#div_id_southlimit_filetype").hide();
-            $("#div_id_westlimit_filetype").hide();
-            $("#div_id_uplimit_filetype").hide();
-            $("#div_id_downlimit_filetype").hide();
+function setFileTypeSpatialCoverageFormFields(logical_type){
```

Comment:
I've already approved this PR. 

Reification becomes extremely important when we crowdsource something.  Then it is really important to list the source of the inference. E.g., "the HydroShare community" thinks this is a useful resource.  

There is a [really good chapter](https://library.oreilly.com/book/9780596002633/practical-rdf/53.xhtml?ref=toc#pracrdf-CHP-4-SECT-3.1) on this in the O'Reilly book "Practical RDF". That link might take you there -- and might not. It also discusses some of the controversy surrounding reification and its pitfalls.